tools/ci/run-examples.sh

```diff
@@ -63,7 +65,19 @@ done
 echo "Stopping standalone Flink cluster."
 ./flink-"${FLINK_VERSION}"/bin/stop-cluster.sh
 
-python -m pip install "${wheel_file}[spark]"
+echo "Downloading Spark."
+curl -LO https://archive.apache.org/dist/spark/spark-"${SPARK_VERSION}"/spark-"${SPARK_VERSION}"-bin-hadoop3.tgz
+tar -xzf spark-"${SPARK_VERSION}"-bin-hadoop3.tgz
+
+# Add localhost to known hosts for starting standalone spark cluster.
+ssh-keygen -t rsa -f "$HOME/.ssh/id_rsa" -P ""
```

Comment:
Can we avoid overwriting users' existing ssh key file?

Reply:
OK, I changed to generate the key with a non-default file name, which can avoid overwriting the existing key.